src/components/templates/product-table/index.tsx

```diff
@@ -548,41 +479,52 @@ const ProductTable: React.FC<ProductTableProps> = () => {
             handleSearch={handleSearch}
             {...getTableProps()}
           >
-            <Table.Head>
-              {headerGroups?.map((headerGroup) => (
-                <Table.HeadRow {...headerGroup.getHeaderGroupProps()}>
-                  {headerGroup.headers.map((col) => (
-                    <Table.HeadCell
-                      className="min-w-[100px]"
-                      {...col.getHeaderProps()}
-                    >
-                      {col.render("Header")}
-                    </Table.HeadCell>
+            {showList ? (
+              <>
+                <Table.Head>
+                  {headerGroups?.map((headerGroup) => (
+                    <Table.HeadRow {...headerGroup.getHeaderGroupProps()}>
+                      {headerGroup.headers.map((col) => (
+                        <Table.HeadCell
+                          className="min-w-[100px]"
+                          {...col.getHeaderProps()}
+                        >
+                          {col.render("Header")}
+                        </Table.HeadCell>
+                      ))}
+                    </Table.HeadRow>
                   ))}
-                </Table.HeadRow>
-              ))}
-            </Table.Head>
-            <Table.Body {...getTableBodyProps()}>
-              {rows.map((row) => {
-                prepareRow(row)
-                return (
-                  <Table.Row
-                    color={"inherit"}
-                    linkTo={`/a/products/${row.original.id}`}
-                    actions={getActionablesForProduct(row.original)}
-                    {...row.getRowProps()}
-                  >
-                    {row.cells.map((cell, index) => {
-                      return (
-                        <Table.Cell {...cell.getCellProps()}>
-                          {cell.render("Cell", { index })}
-                        </Table.Cell>
-                      )
-                    })}
-                  </Table.Row>
-                )
-              })}
-            </Table.Body>
+                </Table.Head>
+                <Table.Body {...getTableBodyProps()}>
+                  {rows.map((row) => {
+                    prepareRow(row)
+                    return (
+                      <Table.Row
```

Comment:
Can we extract this to use the same hooks as the product tile in a component as below?

```javascript 
const ProductRow = ({ row }) => {
  const product = row.original
  const { actions } = useProductActions(product)

  return (
    <Table.Row
      color={"inherit"}
      linkTo={`/a/products/${product.id}`}
      actions={actions}
      {...row.getRowProps()}
    >
      {row.cells.map((cell, index) => {
        return (
          <Table.Cell {...cell.getCellProps()}>
            {cell.render("Cell", { index })}
          </Table.Cell>
        )
      })}
    </Table.Row>
  )
}
```